doc/whats_new.rst

```diff
@@ -70,6 +70,8 @@ BUG
 
     - Fix a bug in :meth:`mne.time_frequency.AverageTFR.plot` when plotting without a colorbar by `Jaakko Leppakangas`_
 
+    - Fix ``_filenames`` attribute in creation of :class:`mne.io.RawArray` with :func:`mne.preporcessing.ica._sources_as_raw` by `Paul Pasler`_
+
```

Comment:
take a look at the bottom of the file, there you can link  `Paul Pasler`_ to your webpage or github account.